pkg/tfshim/shim.go

```diff
@@ -41,8 +41,6 @@ type InstanceDiff interface {
 	ProposedState(res Resource, priorState InstanceState) (InstanceState, error)
 	Destroy() bool
 	RequiresNew() bool
-	EncodeTimeouts(timeouts *ResourceTimeout) error
```

Comment:
These methods are removed in favor of supplying them as options when constructing the InstanceDiff.